subsys/net/lib/azure_iot_hub/src/azure_iot_hub_cloud.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <net/cloud.h>
+#include <net/azure_iot_hub.h>
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(azure_iot_hub_cloud, CONFIG_AZURE_IOT_HUB_LOG_LEVEL);
+
+static struct cloud_backend *azure_iot_hub_backend;
+
+static void api_event_handler(struct azure_iot_hub_evt *evt)
+{
+	struct cloud_backend_config *config = azure_iot_hub_backend->config;
+	struct cloud_event cloud_evt = { 0 };
+
+	switch (evt->type) {
```

Comment:
translation of AZURE_IOT_HUB_EVT_CONNECTING to CLOUD.....EVT_CONNECTING is missing.

Reply:
Nice catch